server/app/services/export/PdfExporter.java

```diff
@@ -126,9 +130,38 @@ private byte[] buildPDF(
         Paragraph time =
             new Paragraph("Answered on : " + date, FontFactory.getFont(FontFactory.HELVETICA, 10));
         time.setAlignment(Paragraph.ALIGN_RIGHT);
+        Paragraph eligibility = new Paragraph();
+        if (isEligibilityEnabledInProgram) {
+          try {
+            Optional<EligibilityDefinition> eligibilityDef =
+                programDefinition.getBlockDefinition(answerData.blockId()).eligibilityDefinition();
+            if (eligibilityDef.isPresent()
+                && eligibilityDef
```

Comment:
**nitpick (blocking):** with `map` and then `orElse(false)` I don't think you need `eligibilityDef.isPresent() &&`?

Reply:
Yes good catch - thanks!